common/src/database/SimulationSaveData.ts

```diff
@@ -7,6 +7,7 @@ import {
 /** Data type for Simulation */
 export type SimulationSaveDataRaw = {
   snapshot: SimulationSnapshot;
+  username?: string;
```

Comment:
This is not optional

Reply:
Oraya optional koymayınca export yaparken type uymuyor, o zaman export için ayrı type yapabilirim

Reply:
ok uğraşma o zaman kalsın